### PR TITLE
Default mapping: <C-n> start with word boundaries, g<C-n> start without

### LINKED ISSUE #210: "No more matches" on single special characters
I can't use multiple cursors on single special characters like ".
Thus, the 3rd example "Do it backwards too! This is not just a replay of the above gif :)" on main page does not work for me. This error is displayed:

> No more matches

I don't have the error for single letters. 

OS : ubuntu 14.04
VIM - Vi IMproved 8.0
iterm, remote ssh

[my .vimrc](https://github.com/terryma/vim-multiple-cursors/files/513385/vimrc.txt)

edit: I reproduced the bug on 
mac os x 
VIM - Vi IMproved 8.0 (installed with homebrew)
iterm, local file

### PR DESCRIPTION
**Pull request by antoinemadec**
Fix #210

### CODE REVIEW

**Comment by antoinemadec** [comment]
@eapache @balta2ar , could you please review it ?
I'd like to have a second opinion before merging it :)

File: doc/multiple_cursors.txt [hunk 1]
```diff
@@ -83,52 +101,16 @@ plugin taking over your favorite key bindings, you can turn off the default
...
-<
-
-In this configuration <C-n> will start multicursor mode using word boundaries
-(but only in Normal mode, as it does not make much sense to use it in Visual
-mode). Old behaviour without word boundaries is still available using
-g<C-n>.
-
-IMPORTANT: Please note that currently only single keystrokes and special
-keys can be mapped. This contraint is also the reason why multikey commands
-such as `ciw` do not work and cause unexpected behavior in Normal mode. This
```
Comment (balta2ar) [inline doc/multiple_cursors.txt]: Sorry, I may have missed it, is this remark about `ciw` not true anymore?  Can I do `ciw` in Normal mode when multiple cursors are active?

**Comment by balta2ar** [comment]
Left one question but other than that no objections from my side, I've had such mapping in my vimrc for ages (and that's the default behavior in other editors).

**Comment by eapache** [comment]
fine by me

File: doc/multiple_cursors.txt (hunk 1 above)
Comment (antoinemadec) [reply doc/multiple_cursors.txt]: @balta2ar , yes, ```ciw``` works in normal mode.
Make sure to have the following in your **.vimrc**:
```vim
let g:multi_cursor_exit_from_insert_mode = 0
let g:multi_cursor_exit_from_visual_mode = 0
```
Then, ```<C-n>...<C-n><Esc>ciw``` will work fine :)

**Pull request merged**